web/html/src/components/table/Table.tsx

```diff
@@ -42,6 +42,9 @@ type TableProps = {
   /** enables item selection */
   selectable: boolean;
 
+  /** tells if a row is selectable */
+  isSelectEnabled?: (row: any) => boolean;
```

Comment:
I would recommend merging this into one prop with `selectable` similar to `deletable` since there is no reason to set `selectable = false` but still give a value for this prop.

Reply:
👍🏻